src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -611,6 +611,8 @@ bool program_node::is_padded_spatial(size_t idx) const {
     auto& layout = get_output_layout(idx);
     const auto& lower_size = layout.data_padding._lower_size;
     const auto& upper_size = layout.data_padding._upper_size;
+    if (layout.get_spatial_rank() < 3)
+        return false;
     return std::any_of(std::begin(lower_size) + 2, std::begin(lower_size) + layout.get_spatial_rank() - 1,
```

Comment:
Maybe problem is in the range itself, probably meant:
```suggestion
    return std::any_of(std::begin(lower_size) + 2, std::begin(lower_size) + 2 + layout.get_spatial_rank() - 1,
```
And it can be reduced to
```suggestion
    return std::any_of(std::begin(lower_size) + 2, std::end(lower_size),
```